doc/source/user_guide/timeseries.rst

```diff
@@ -1468,11 +1468,16 @@ or some other non-observed day.  Defined observance rules are:
     :header: "Rule", "Description"
     :widths: 15, 70
 
+    "next_workday", "move Saturday and Sunday to Monday"
+    "previous_workday", "move Saturday and Sunday to Friday"
```

Comment:
This one isn't *quite* right ... it moves to the previous workday even if the input is a weekday, not just for Sat and Sun (as opposed to what previous_friday does)